documentation/src/main/markdown/integrations/azureplugin/azurepluginreleasenotes.md

```diff
@@ -1,5 +1,17 @@
 # Release Notes for Azure DevOps Plugin
 
+## Version 10.1.0
+**New features**
+
+* (DETECTADO-102) Added support for Node16 and Node20_1 Node.js versions which will remove warnings for Node10 end of life.
```

Comment:
Suggestion:
* (DETECTADO-102) Added support for Node.js versions Node16 and Node20_1. Node10 end of life warning will no longer be displayed.